### ISSUE TITLE
erase-install timing out and not moving on 

### ISSUE THREAD

**Issue by GabeShack** [grahampugh/erase-install#241]
**Describe the bug**
Yesterday and today we have seen the script fail for groups of computers.
 
**To Reproduce**
 - If applicable, please provide the full command used when running the script.
 - Please describe how the script was run. [e.g. Standalone via Terminal, in Jamf Policy from the uploaded pkg, in Jamf policy from uploaded script only, via another MDM provider]
This is being run via a policy in Jamf by calling a policy trigger specific to "Erase and Install" when then installs the current erase-install package, then runs the following command: 
softwareupdate --reset-ignored & /Library/Management/erase-install/erase-install.sh --erase --version=12.3.1 --current-user --check-power --overwrite --move
**Expected behavior**
WE normally see this give the message that the installer is being downloaded, then it will go to a full screen message about running the erase.
**Code/log output**
Please supply the full command used, and if applicable, add full output from Terminal or from `/var/log/erase-install.log`. Either upload the log, or paste the output in a code block (triple backticks at the start and end of the code block please!).
the command running is as follows:
```softwareupdate --reset-ignored & /Library/Management/erase-install/erase-install.sh --erase --version=12.3.1 --current-user --check-power --overwrite --move```

log file for failure:
 ```[erase-install] v26.0 script execution started: Mon Apr  4 13:18:38 EDT 2022
   [erase-install] Caffeinating this script (pid=1216)
   [check_free_space] OK - 204 GB free/purgeable disk space detected
   [check_power_status] OK - AC power detected
   [erase-install] Looking for existing installer app or pkg
   [find_existing_installer] No valid installer found.
yes macadmin is a member of everyone
   [get_user_details] macadmin is not a Volume Owner
button returned:OK

   [erase-install] attempting to terminate the 'caffeinate' process - Termination message indicates success
/Library/Management/erase-install/erase-install.sh: line 1042:  1233 Terminated: 15          /usr/bin/caffeinate -dimsu -w $$
```

**Screenshots**

**Environment (please complete the following information):**
 - OS version [e.g. 12.0.1] We are running these on machines with 11.6 fresh out of the box to get them to 12.3.1
 - erase-install version [e.g. 24.0] - please upgrade to the latest version before submitting a bug report.
Erase-install version 26
**Additional context**
Im wondering if we are seeing a problem with timing on the creation of the macadmin user where it isnt showing as a volume owner even though its the only real admin user on the device besides the jamf management user.  Since it still is prompting for the macadmin password to continue to the erase, id assume it thinks they are a volume owner.

**Comment by grahampugh** [comment]
The macadmin user is not a Volume Owner so the script has stopped. And I do not see any evidence in the log that a password was asked for. 

(But it doesn't look like the complete log because the line that shows version is missing, so I don't know if you removed any other lines.)

**Comment by GabeShack** [comment]
```   [erase-install] Caffeinating this script (pid=1954)
   [check_free_space] OK - 204 GB free/purgeable disk space detected
   [check_power_status] OK - AC power detected
   [erase-install] Looking for existing installer app or pkg
   [find_existing_installer] No valid installer found.
yes macadmin is a member of everyone
   [get_user_details] macadmin is a Volume Owner
   [check_password] Success: the password entered is the correct login password for macadmin.
yes macadmin is a member of admin
   [erase-install] Opening jamfHelper download message (language=en)
   [get_installinstallmacos] installinstallmacos.py is in /Library/Management/erase-install
   [get_relocatable_python] Relocatable Python is installed in /Library/Management/erase-install
   [run_installinstallmacos] Checking that selected version 12.3.1 is available

   [run_installinstallmacos] This command is now being run:

   installinstallmacos.py --workdir /Library/Management/erase-install --ignore-cache --raw --version 12.3.1 --validate
/Library/Management/erase-install/erase-install.sh: line 1207: /Library/Management/erase-install/Python.framework/Versions/Current/bin/python3: Bad CPU type in executable
   [run_installinstallmacos] Error obtaining valid installer. Cannot continue.


   [erase-install] attempting to terminate the 'caffeinate' process - Termination message indicates success
/Library/Management/erase-install/erase-install.sh: line 1042:  1969 Terminated: 15          /usr/bin/caffeinate -dimsu -w $$```

**Comment by grahampugh** [comment]
Ok so that's a completely different log. You need to install Rosetta to use the python that comes with v26.0.

**Comment by GabeShack** [comment]
Thank you that make more sense, as we stopped pushing the first run installers for anything older than MacOS 12.

**Comment by grahampugh** [comment]
FYI 26.1 pre-release is theoretically universal, though I've never actually tested it on a non-Rosetta device as Rosetta is the first thing we deploy to all M1s. Feel free to try it out 😃

**Comment by GabeShack** [comment]
I can try that since we have over 100 more m1s to set up that shipped with 11.0.  I'll report back what we find.
